pkg/scanner/scan_test.go

```diff
@@ -196,6 +198,7 @@ func TestScanner_ScanArtifact(t *testing.T) {
 			mockArtifact.ApplyInspectExpectation(tt.inspectExpectation)
 
 			s := NewScanner(d, mockArtifact)
+			clock.SetFakeTime(t, time.Date(1, time.January, 1, 0, 0, 0, 0, time.UTC))
```

Comment:
can you add `StartTime` and `EndTime` to [testcases](https://github.com/aquasecurity/trivy/blob/11203d83a3beba5499a8170a0aa8aaf17303e84f/pkg/scanner/scan_test.go#L99). 
Because it can confusing that testcases don't have these fields.